.gitignore

```diff
@@ -10,3 +10,4 @@
 .env
 *.log
 *.swp
+.idea
```

Comment:
This should be in your global git ignore, not added to repo specific git ignores

Reply:
Oh yea, I forgot that existed.